LPS-107707 Initialize localizable text field attributes

carolmariaabb opened the pull request.

timpak committed:
LRQA-53687 Add additional configuration options for Similar Results

timpak committed:
LRQA-53687 Add additional paths for Similar Results

timpak committed:
LRQA-53687 Add tests to configure Maximum Item Display in Similar Results

BryanEngler committed:
LPS-104938 portal-search-admin-web: Move connection info from Index Actions page to a new Search Engine page

https://issues.liferay.com/browse/LPS-104938

BryanEngler committed:
LPS-104938 portal-search-solr7-impl: Implement getConnectionInformationList

https://issues.liferay.com/browse/LPS-104938

BryanEngler committed:
LPS-104938 portal-search-elasticsearch6-impl: Dont implement getConnectionInformationList for ES6 in master

https://issues.liferay.com/browse/LPS-104938

BryanEngler committed:
LPS-104938 portal-search-api: Add interfaces for holding connection/node information

https://issues.liferay.com/browse/LPS-104938

BryanEngler committed:
LPS-104938 portal-search-admin-web: buildlang

https://issues.liferay.com/browse/LPS-104938

BryanEngler committed:
LPS-104938 portal-search-elasticsearch7-impl: Implement getConnectionInformationList

https://issues.liferay.com/browse/LPS-104938

thektan committed:
LPS-104938 portal-search-admin-web: Update css

https://issues.liferay.com/browse/LPS-104938

thektan committed:
LPS-104938 portal-search-admin-web: Remove unused css

https://issues.liferay.com/browse/LPS-104938

thektan committed:
LPS-104938 portal-search-admin-web: Update reindex page refresh

Since index actions is not the default tab, when refreshing the page we need to pass in the specific tab id "index-actions".

https://issues.liferay.com/browse/LPS-104938

Commit:
LPS-104938 LRQA-54832 Update Index Actions tab

Commit:
LPS-104938 LRQA-54832 Add a step to goto search product menu

Commit:
LPS-104938 LRQA-54832 Add locators of search client version and engine vendor

Commit:
LPS-104938 LRQA-54832 Add condition of engine vendor and versions

Commit:
LPS-104938 LRQA-54832 Add condition for getting engine vendor and versions

marcelabc committed:
LPS-103901 Change field icon and display order

marcelabc committed:
LPS-103901 update language terms

marcelabc committed:
LPS-103901 Set view mode in order to display the form instance record

marcelabc committed:
LPS-103901 Creation of a new field: geolocation

marcelabc committed:
LPS-103901 Add dependencies

marcelabc committed:
LPS-103901 Geolocation component

timpak committed:
LRQA-55133 Add tests to configure Stop Words in Similar Results

timpak committed:
LRQA-55127 Add tests to configure Document Frequency in Similar Results

timpak committed:
LRQA-55129 Add tests to configure Word Length in Similar Results

timpak committed:
LRQA-55349 Add tests to configure Term Frequency in Similar Results

timpak committed:
LRQA-55373 Add tests to configure Maximum Query Terms in Similar Results

timpak committed:
LRQA-55130 Add tests to configure Minimum Should Match in Similar Results

timpak committed:
LRQA-55375 Add tests to configure Fields in Similar Results

timpak committed:
LRQA-55547 Set Fields for all Similar Results tests

timpak committed:
LRQA-55547 Don't skip portal instance unless it's needed

timpak committed:
LRQA-55292 Add coverage for wiki pages

timpak committed:
LRQA-55376 Add workaround

timpak committed:
LRQA-55390 Ignore test

Commit:
LPS-107707 Create Localizable Text field type

Commit:
LPS-103901 - Updating Form.ADD_FIELD_TYPE path and adding addTextField macro

Commit:
LPS-103901 - Replacing gotoAddField by addTextField in all the scripts

Commit:
LPS-107707 Add constant for new field type

Commit:
LPS-107707 Implement localization component

Commit:
LPS-107707 Adds SF suppressions

BryanEngler committed:
LPS-104938 portal-search: Implement interfaces

https://issues.liferay.com/browse/LPS-104938

lipusz committed:
LPS-107863 portal-search-elasticsearch7-impl: Bump "netty-all" to 4.1.36.Final to fix https://github.com/netty/netty/pull/8564

4.1.36.Final is used by Elasticsearch 7.3.0 : https://github.com/elastic/elasticsearch/blob/v7.3.0/buildSrc/version.properties#L23

Our current client version is 7.3.0 so let's go with what Elasticsearch is also using.

https://issues.liferay.com/browse/LPS-107863

lipusz committed:
LPS-107863 portal-search-elasticsearch7-impl: Do not import some new packages brought in by the "netty-all" bump

Otherwise we get

2020-01-30 01:12:01.868 ERROR [fileinstall-/home/dev/liferay/bundles/7.2.x/osgi/portal][LogService:93] Error while starting bundle: file:/home/dev/liferay/bundles/7.2.x/osgi/portal/com.liferay.portal.search.elasticsearch7.impl.jar
org.osgi.framework.BundleException: Could not resolve module: com.liferay.portal.search.elasticsearch7.impl [240]_  Unresolved requirement: Import-Package: com.oracle.svm.core.annotate_ [Sanitized]
	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1297)
	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1270)
	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1259)
	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:519)
	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:369)
	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:320)

2020-01-30 01:16:02.979 ERROR [fileinstall-/home/dev/liferay/bundles/7.2.x/osgi/portal][LogService:93] Error while starting bundle: file:/home/dev/liferay/bundles/7.2.x/osgi/portal/com.liferay.portal.search.elasticsearch7.impl.jar
org.osgi.framework.BundleException: Could not resolve module: com.liferay.portal.search.elasticsearch7.impl [240]_  Unresolved requirement: Import-Package: io.netty.internal.tcnative_ [Sanitized]
	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundle(DirectoryWatcher.java:1297)
	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startBundles(DirectoryWatcher.java:1270)
	at org.apache.felix.fileinstall.internal.DirectoryWatcher.startAllBundles(DirectoryWatcher.java:1259)
	at org.apache.felix.fileinstall.internal.DirectoryWatcher.doProcess(DirectoryWatcher.java:519)
	at org.apache.felix.fileinstall.internal.DirectoryWatcher.process(DirectoryWatcher.java:369)
	at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:320)

https://issues.liferay.com/browse/LPS-107863

Commit:
LPS-108013 deal with repeatable values

Commit:
LPS-108013 enable translation manager

Commit:
LPS-108013 update packageinfo

Commit:
LPS-108013 get available locale from translation manager

4lejandrito committed:
LPS-107877 Adding renderer provider interface

4lejandrito committed:
LPS-107877 Using the provider inside item selector

4lejandrito committed:
LPS-107877 Providing a default implementation with the original logic

4lejandrito committed:
LPS-107877 We need to override this reference

4lejandrito committed:
LPS-107877 Providing a custom renderer for documents and media

4lejandrito committed:
LPS-107877 Renaming

AliciaGarciaGarcia committed:
LPS-107877 baseline

Commit:
LPS-103901 Replacing Text Field by Text

Commit:
LPS-103901 Replacing Text Field by Text at ViewBlankLabelWhenAddingNewField test case

Commit:
LPS-103901 Replacing Text Field by Text at ViewDefaultFieldNamesWithPunctuationCharacters test case

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

Commit:
LPS-103901 SF

sandrodw3 committed:
LPS-106025 Renamed Columns to Row

Commit:
LPS-108013 define properties

Commit:
LPS-108013 pass available language ids

AliciaGarciaGarcia committed:
LPS-107877 Show application disabled on page

AliciaGarciaGarcia committed:
LPS-107877 Show personalized message and let back to select on web

AliciaGarciaGarcia committed:
LPS-107877 BuildLang

Commit:
LPS-108013 get default data layout id

pavel-savinov committed:
LPS-107012 Add missing field to support staging for the entity

pavel-savinov committed:
LPS-107012 Set company id from group and fallback to service context when group doesn't exist

pavel-savinov committed:
LPS-107012 Regenerate service

pavel-savinov committed:
LPS-107012 Upgrade to set companyId for existing classed model usages

robertoDiaz committed:
LPS-107728 Expose a method in order to allow implementations obtain the order by

robertoDiaz committed:
LPS-107728 Use the method

sandrodw3 committed:
LPS-106025 Recalculate decorators when fragment content changes

carolmariaabb committed:
LPS-107707 Implement the back-end side associated with the Text Input Localizable

Remove localizable text form field from forms sidebar

Commit:
LPS-108013 fix styling

inacionery committed:
LPS-102664 | LPS-107807 Add WorkflowTaskInstanceIdComparator

inacionery committed:
LPS-102664 | LPS-107807 Add Kaleo instance ID as a sortable column

inacionery committed:
LPS-102664 | LPS-107807 Use sort on getWorkflowTasksPage

inacionery committed:
LPS-102664 | LPS-107807 Add dependency

inacionery committed:
LPS-102664 | LPS-107807 Bump

drewbrokke committed:
LPS-107779 account-rest-impl - schema - adds POST endpoint

drewbrokke committed:
LPS-107779 account - build rest

drewbrokke committed:
LPS-107779 account-rest-test - implements test helper methods

drewbrokke committed:
LPS-107779 account-rest-impl - overrides and implements postAccount method

drewbrokke committed:
LPS-107779 account-service - makes name required when creating an AccountEntry

drewbrokke committed:
LPS-107779 account-service - apply to updateAccountEntry

drewbrokke committed:
LPS-107779 account-rest-impl - adds AccountEntryNameExceptionMapper

drewbrokke committed:
LPS-107779 account-rest-impl - schema - makes status writable

drewbrokke committed:
LPS-107779 account - build rest

oliv-yu committed:
LPS-107815 portal-search-web: Check for null in searchbar ADT

petershin committed:
LPS-108031 Add property to enable Gradle build scans

timpak committed:
LRQA-55547 Remove redundant assertions

timpak committed:
LRQA-55547 Add macro to setup Similar Results page

timpak committed:
LRQA-55547 Apply

sandrodw3 committed:
LPS-106025 Logic to determine whether editables decoration should be shown or not moved to function

pavel-savinov committed:
LPS-107012 Basic staged model data handler and repository for the classed model usage

pavel-savinov committed:
LPS-107012 Staged model repository methods

pavel-savinov committed:
LPS-107012 Export layout classed model usage and its dependency

pavel-savinov committed:
LPS-107012 Import layout classed model usage and its dependency

sandrodw3 committed:
LPS-106025 Show decoration when at least one editable is translated or mapped

pavel-savinov committed:
LPS-107012 This check makes sense only for the content pages

pavel-savinov committed:
LPS-107012 Export associated layout classed model usages

AliciaGarciaGarcia committed:
LPS-104145 Create new DDM editor configuration for DL

AliciaGarciaGarcia committed:
LPS-104145 baseLine

pavel-savinov committed:
LPS-107012 Import associated layout classed model usages

pavel-savinov committed:
LPS-107012 Add XStream configurator for layout classed model usages

drewbrokke committed:
LPS-107782 account-rest-test - SF - uses the tearDown pattern found in OrganizationResourceTest

adolfopa committed:
LPS-107728 SF

Commit:
LPS-108013 fix tests

adolfopa committed:
LPS-104145 Move to internal package

petershin committed:
LPS-108031 Use boolean flag gradle.build.scan.enabled

petershin committed:
LPS-108031 Enable scans for CI

petershin committed:
LPS-108031 Add property for additional tag

Preston-Crary committed:
LPS-107977 Versioning.

carolmariaabb committed:
LPS-107707 Implement the front-end side associated with the Text Input Localizable

mtambara committed:
LPS-107983 Only strip separator when it exists

ivicac committed:
LPS-107626 - batch-engine-api - Adjust BatchEngineTaskItemDelegate to be able to be used by REST Builder

ivicac committed:
LPS-107626 - batch-engine-service - Update logic to use new BatchEngineTaskItemDelegate

ivicac committed:
LPS-107626 - batch-engine-test - Updatetests

ivicac committed:
LPS-107626 - headless-batch-engine-impl - Propgate path prameters to batch execution

hhuijser committed:
LPS-105380 SF, buy space

hhuijser committed:
LPS-105380 SF, buy space

hhuijser committed:
LPS-105380 SF

shuyangzhou committed:
LPS-108185 Drop vm support for sql translation, we don't do this any more.

shuyangzhou committed:
LPS-108185 Deprecated evaluate support and remove all usages.

hhuijser committed:
LPS-107918 Change headers to match name of check

hhuijser committed:
LPS-107918 No need to pass markdown files. SF is smart enough to find em

hhuijser committed:
LPS-107918 Move markdowns to new folder (leaving the old ones as well to make sure people using older version of SF are still pointed to existing files)

hhuijser committed:
LPS-107918 Rename to match check name

hhuijser committed:
LPS-107918 Change headers to match name of check

hhuijser committed:
LPS-107918 Move markdowns to new folder (leaving the old ones as well to make sure people using older version of SF are still pointed to existing files)

hhuijser committed:
LPS-107918 Rename to match check name

hhuijser committed:
LPS-107918 Change headers to match name of check

hhuijser committed:
LPS-107918 Move markdowns to new folder (leaving the old ones as well to make sure people using older version of SF are still pointed to existing files)

hhuijser committed:
LPS-107918 Rename to match check name

hhuijser committed:
LPS-107918 Change headers to match name of check

hhuijser committed:
LPS-107918 Move markdowns to new folder (leaving the old ones as well to make sure people using older version of SF are still pointed to existing files)

hhuijser committed:
LPS-107918 Rename to match check name

hhuijser committed:
LPS-107918 Change headers to match name of check

hhuijser committed:
LPS-107918 No need to pass markdown files. SF is smart enough to find em

hhuijser committed:
LPS-107918 Move bnd_bundle_information.markdown

hhuijser committed:
LPS-107918 Spit up logic. Two checks should not use the same markdown

hhuijser committed:
LPS-107918 No need to pass markdown files

jkappler committed:
LPS-107942 Remove max number of feeds and expanded entries per feed

adolfopa committed:
LPS-107728 Semver

adolfopa committed:
LPS-107877 Change API from provider to customizer

Using a factory/provider here has a bunch of problems:

- The ItemSelectorViewRendererImpl class is private to the module, so providers
  need to reimplement everything, even if they want to customize some little
  detail.

- There cannot be more than one implementation at the same time. There could be
  multiple implementations that need to customize some details (e.g. inject some
  stuff in the request). That is impossible to implement with the provider.

- Messier API. We need to expose the ItemSelectorViewRendererImpl constructor
  signature (an implementation detail) as part of the public API.

Using a customizer, multiple implementations can be chained, and they may choose
to delegate part of the logic into the original impl, or ignore it
altogether. The API is also simpler to implement.

jkappler committed:
LPS-107942 Correct attribute

jkappler committed:
LPS-107942 Correct parameter

jkappler committed:
LPS-107942 We don't use AUI.$ anymore, so use CollapseProvider instead

p2kmgcl committed:
LPS-106025 SF move non-component functions to the bottom of the file

ealonso committed:
LPS-108126 Adds new finder to find the default templates for a type

ealonso committed:
LPS-108126 Auto

ealonso committed:
LPS-108126 baseline

ealonso committed:
LPS-108126 Adds new method

markocikos committed:
LPS-107869 Update CKEditor version to 4.13.1

p2kmgcl committed:
LPS-106025 Unconditionally render editable decorations

ealonso committed:
LPS-108126 Language key

ealonso committed:
LPS-108126 buildLang

ealonso committed:
LPS-108126 Mark default master page for widget pages

ealonso committed:
LPS-108126 Allow users to mark a master page as default for widget pages

ealonso committed:
LPS-108126 Take into account the default master page when we are creating widget pages

Commit:
LPS-107015 Fix dropdown menu icon in westeros theme

adolfopa committed:
LPS-107877 Register customizers by criterion

p2kmgcl committed:
LPS-106025 Compute editable decorations state via selectors instead of new hooks

adolfopa committed:
LPS-107877 SF - Reduce nesting

adolfopa committed:
LPS-107877 SF

ealonso committed:
LPS-108126 Adds custom mark as defualt action for blank master page

adolfopa committed:
LPS-107877 SF

jbalsas committed:
LPS-107869 Regenes yarn.lock file (auto-generated)

adolfopa committed:
LPS-107877 Build lang

p2kmgcl committed:
LPS-106025 Handle mouse events from editable decoration

brianchandotcom committed:
LPS-108013 SF

brianchandotcom committed:
LPS-108013 Bump for 7.2.x

mtambara committed:
LPS-108159 Move out Sybase-specific logic to SybaseDB

wincent committed:
LPS-105147 Apply Prettier formatting overrides in liferay-portal

Via the new version of liferay-npm-scripts:

https://github.com/liferay/liferay-npm-tools/pull/365

ealonso committed:
LPS-108114 SF

shuyangzhou committed:
LPS-108159 Prevent unnecessary processing

brianchandotcom committed:
LPS-108126 SF

shuyangzhou committed:
LPS-107626 SF

hhuijser committed:
LPS-105380 Fix JavaSourceProcessorTest

shuyangzhou committed:
LPS-107626 Prefer parameters over fields, less states.

shuyangzhou committed:
LPS-107983 entryPathString is never empty

brianchandotcom committed:
LPS-107782 won't work without this

mtambara committed:
LPS-108159 Add test

marcelabc committed:
LPS-103901 buildLang

hhuijser committed:
LPS-107918 Generated

hhuijser committed:
LPS-103975 baseline

hhuijser committed:
LPS-107918 Move logic to its own check

hhuijser committed:
LPS-107918 Move logic to its own check

hhuijser committed:
LPS-107918 Remove logic to check for chaining, we'll do that using checkstyle soon

thektan committed:
LPS-107815 portal-search-web: Prevent js breaking from undefined vars

hhuijser committed:
LPS-107918 Move markdown to new folder (leaving the old one as well to make sure people using older version of SF are still pointed to existing files)

hhuijser committed:
LPS-107918 Rename to match check name

hhuijser committed:
LPS-107918 Change header to match name of check

drewbrokke committed:
LPS-107779 account-rest-impl - schema - adds missing '@review' annotation

drewbrokke committed:
LPS-107779 account-rest-impl - AccountEntryNameExceptionMapper -> AccountNameExceptionMapper. Follows headless entity rather than service builder

hhuijser committed:
LPS-105380 Fix NPE (when class has imports with star)

Commit:
LPS-107916 Add null to values list when no value is found for the field

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/analytics/.gitrepo'.

hhuijser committed:
LPS-107918 Move constant to SourceFormatterUtil

hhuijser committed:
LPS-107918 If checkstyle check has property 'documentationLocation' defined, use that when showing message

hhuijser committed:
LPS-107918 Rename method, no logical changes

hhuijser committed:
LPS-107918 Move markdowns to new folder (leaving the old ones as well to make sure people using older version of SF are still pointed to existing files)

hhuijser committed:
LPS-107918 Rename to match check name

brianchandotcom committed:
LRAC-4949 remove

brianchandotcom committed:
LRAC-4949 for master

hhuijser committed:
LPS-107918 Change headers to match name of check

hhuijser committed:
LPS-107918 Remove urls from message in checkstyle. SF is smart enough to find em

brianchandotcom committed:
LRAC-4949 baseline

hhuijser committed:
LPS-107918 Simplify

hhuijser committed:
LPS-107918 Move markdown to new folder (leaving the old one as well to make sure people using older version of SF are still pointed to existing files)

hhuijser committed:
LPS-107918 Rename to match check name

hhuijser committed:
LPS-107918 Change header to match name of check

hhuijser committed:
LPS-107918 Remove urls from message in checkstyle

brianchandotcom committed:
LPS-107779 Compare with modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/jaxrs/exception/mapper/NoSuchDataRecordCollectionExceptionMapper.java

brianchandotcom committed:
LPS-107779 FOr now, match DocumentFileNameExceptionMapper because AccountEntryLocalServiceImpl throws AccountEntryNameException during add AND update.. so your message will not be accurate as soon as we introduce patch/put

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as 2ced9fdeb5d8042d4b9411ab829c66565d0e71c6

brianchandotcom committed:
LPS-107012 as used

hhuijser committed:
LPS-107918 Move markdowns to new folder (leaving the old ones as well to make sure people using older version of SF are still pointed to existing files)

hhuijser committed:
LPS-107918 Merge files with information regarding the same check

BryanEngler committed:
LPS-104938 portal-search-admin-web: Update tests

https://issues.liferay.com/browse/LPS-104938

joshchong committed:
LPS-104938 LRQA-54832 Use more explicit assertions to account for the variations in UI

hhuijser committed:
LPS-107918 No need to pass markdown files. SF is smart enough to find em

shuyangzhou committed:
LPS-108185 Semantic versioning

CAustin582 committed:
LPS-107965 Flip logic; thirdPartyCheckedIdentifiers is a whitelist, not a blacklist

carolmariaabb committed:
LPS-107707 Add frontend tests for localizable text field type

hhuijser committed:
LPS-107918 Remove unused constructors

carolmariaabb committed:
LPS-107707 Tests snapshots

hhuijser committed:
LPS-107918 Fix NPE

hhuijser committed:
LPS-107918 When parsing fails, add 'JavaClassParser' to the message to avoid confusion

Commit:
LPS-108175 Update LMDD to 05-02-2020

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/analytics/.gitrepo'.

hhuijser committed:
LPS-107918 Refactor, retrieve documentationURLString on the checkstyle/sourcecheck side instead of inside class SourceFormatterMessage

hhuijser committed:
LPS-107918 Move markdown to new folder (leaving the old one as well to make sure people using older version of SF are still pointed to existing files)

hhuijser committed:
LPS-107918 Rename to match check name

hhuijser committed:
LPS-107918 Change headers to match name of check

hhuijser committed:
LPS-107918 Remove urls from message in checkstyle

hhuijser committed:
LPS-107918 No need to pass markdown files. SF is smart enough to find em

hhuijser committed:
LPS-107918 Remove unused logic, nobody is passing markdown file names anymore

hhuijser committed:
LPS-107918 Generated

p2kmgcl committed:
LPS-108186 Include empty permissions in PageStructureSidebar tests

brianchandotcom committed:
LPS-104938 Hugo SF, consistency

brianchandotcom committed:
LPS-104938 We can sort this here since it's an if else statement and not logical (and if we ever change the display order, this won't look out of place). It also matches modules/apps/portal-search/portal-search-admin-web/src/main/resources/META-INF/resources/css/main.scss

.portlet-search-admin {
	@import 'field_mappings';
	@import 'index_actions';
	@import 'search_engine';
}

brianchandotcom committed:
LPS-104938 prevent major version change in bnd.bnd by using ProviderType (and I think that is what it is)

adolfopa committed:
LPS-107877 Move interface to same package as view renderer

adolfopa committed:
LPS-107877 Make field static

adolfopa committed:
LPS-107877 Auto SF

brianchandotcom committed:
LPS-107918 prep next

brianchandotcom committed:
LPS-107918 prep next

pavel-savinov committed:
LPS-107012 Use base company id upgrade instead of setting company id manually
